src/components/works/works.module.scss

```diff
@@ -0,0 +1,152 @@
+.worksWrappper {
+	background-image: url("/backgroundWorks.jpg");
+	background-size: cover;
+	background-position: center;
+	background-repeat: no-repeat;
+}
+
+.worksContainer {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin: 0 auto;
+
+	&__headerWrapper {
+		width: 93.75vw;
+		height: 190px;
+	}
+
+	&__titleImage {
+		width: 89.75vw;
+		margin: 316px 0 410px;
+		max-height: 454px;
+	}
+}
+
+.rectangle {
+	margin-top: auto;
+	width: 84.85%;
+	height: 70px;
+	background-color: var(--floral-white);
+	border-radius: 62px 62px 0 0;
+	flex-shrink: 0;
+}
+
+@media (max-width: 1921px) {
+	.worksContainer {
+		&__headerWrapper {
+			font-size: 5px;
+			width: 91.7vw;
+			height: 142px;
+		}
+
+		&__titleImage {
+			width: 89.8vw;
+			margin: 238px 0 290px;
+			max-height: 341px;
+		}
+	}
+
+	.rectangle {
+		width: 91.2%;
+		border-radius: 50px 50px 0 0;
+	}
+}
+
+@media (max-width: 1441px) {
+	.worksContainer {
```

Comment:
display: block;

can you add this line here and in other similar places where css selector is empty?